### PR TITLE
Controlled cube

### PR DESCRIPTION
**Pull request by ketoni**

### CODE REVIEW

**Review by tkln — COMMENTED** [review]
wat

**Review by tkln — CHANGES_REQUESTED** [review]
move to demo

File: demo/CubeDemo/MotionComponent.cpp [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#include "MotionComponent.hpp"
```
Comment (tkln) [inline demo/CubeDemo/MotionComponent.cpp]: watwatwat

**Review by tkln — CHANGES_REQUESTED** [review]

File: include/Logic/ControlVisitor.hpp [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 #include "Core/Visitor.hpp"
 #include "Logic/TransformComponent.hpp"
-#include "ControlMapComponent.hpp"
+#include "Logic/ControlMapComponent.hpp"
```
Comment (tkln) [inline include/Logic/ControlVisitor.hpp]: Don't move the controls.

**Review by tkln — COMMENTED** [review]
pls fix

**Comment by ketoni** [comment]
ookoo

**Pull request merged**